packages/ui/lib/components/icon/svg/IconSuccess.tsx

```diff
@@ -4,7 +4,7 @@ const IconSuccess = (props: SvgIconProps) => (
   <SvgIcon viewBox="0 0 48 48" {...props}>
     <path
       d="M0 24C0 10.7452 10.7452 0 24 0C37.2548 0 48 10.7452 48 24C48 37.2548 37.2548 48 24 48C10.7452 48 0 37.2548 0 24Z"
-      fill="currentColor"
+      fill="#E6F9EF"
```

Comment:
why can't we get the value to work dynamically anymore? Is there some impediment or blocking issue?